configure

```diff
@@ -576,7 +576,8 @@ def get_version_helper(cc, regexp):
 
 def get_llvm_version(cc):
   return get_version_helper(
-      cc, r"(^clang version|based on LLVM) ([3-9]\.[0-9]+)")
+      cc, r"(^clang version|^FreeBSD clang version|based on LLVM) " +
+      "([3-9]\.[0-9]+)")
```

Comment:
I thought it could be fixed but have no full confirmation if it has no side effects. This is very conservative fix not to break existing checks.